python/main.py

```diff
@@ -0,0 +1,56 @@
+# !/path/to/bin/of/python
+#↑Fill in as appropriate
+
+from abc import ABCMeta, abstractmethod #抽象クラス用
+
+class PartsBehavior(metaclass=ABCMeta):
+    @abstractmethod
+    def getResistance(self):
+        pass
+
+class Cable(PartsBehavior):
+    pass
+
+class Parts(Cable):
+    pass
+
+class Resistance(Parts):
+    def __init__(self, ohm):
+        self.ohm = ohm
+
+    def getResistance(self):
+        return self.ohm
+
+class SerialCable(Cable):
+    def __init__(self, c):
```

Comment:
I think it is preferable for defining a default value as `[]`. And the variable name `c` makes a little bit confused.